ruby/reek/default.yml

```diff
@@ -143,4 +144,4 @@ detectors:
   UtilityFunction:
     enabled: true
     exclude: []
-    public_methods_only: false
\ No newline at end of file
+    public_methods_only: false
```

Comment:
There is no change here. Do you want to change to `public_methods_only: true`?
It may be acceptable but I think we often use functional method that could be extract in a module.
For example, it is preferable to extract method that serialize or build an object because it is a specific responsibility.

UtilityFunction shows that there is a mix of responsibilities.

Reply:
Well, this one is not working properly. e.g Using a map on an instance variable and then using a method inside will trigger this cop. There is an issue open for a while on reek repository for that but currently this is a major bug IMO.